kernel/kernel-benchmarks/src/test/resources/workload_specs/basic_catalog_managed/delta/_delta_log/00000000000000000000.json

```diff
@@ -0,0 +1,4 @@
+{"commitInfo":{"inCommitTimestamp":1712091396253,"timestamp":1712091396253,"operation":"WRITE","operationParameters":{"mode":"ErrorIfExists","partitionBy":"[]"},"isolationLevel":"Serializable","isBlindAppend":true,"operationMetrics":{"numFiles":"1","numOutputRows":"3","numOutputBytes":"996"},"engineInfo":"Apache-Spark/3.5.1 Delta-Lake/3.1.0","txnId":"5df7dc20-b980-4207-a5fd-b69cb4541b2e"}}
+{"metaData":{"id":"2a1e618f-d92a-4c94-bb06-a2808f8b39f3","format":{"provider":"parquet","options":{}},"schemaString":"{\"type\":\"struct\",\"fields\":[{\"name\":\"letter\",\"type\":\"string\",\"nullable\":true,\"metadata\":{}},{\"name\":\"number\",\"type\":\"long\",\"nullable\":true,\"metadata\":{}},{\"name\":\"a_float\",\"type\":\"double\",\"nullable\":true,\"metadata\":{}}]}","partitionColumns":[],"configuration":{"delta.enableInCommitTimestamps":"true","ucTableId":"12345678-1234-1234-1234-123456789abc"},"createdTime":1712091393302}}
```

Comment:
Depending on when this gets merged, we may want to rename 
- `ucTableId` to `catalogManaged.unityCatalog.tableId`
- `catalogOwned-preview` to `catalogManaged`

Please keep me in sync on when you want this merged. Relevant PRs:
- https://github.com/delta-io/delta/pull/5478
- https://github.com/delta-io/delta/pull/5463